core-dictionary/src/main/java/org/apache/kylin/dict/DictionaryGenerator.java

```diff
@@ -109,6 +109,7 @@ public static Dictionary mergeDictionaries(DataType dataType, List<DictionaryInf
             @Nullable
             @Override
             public Dictionary<String> apply(@Nullable DictionaryInfo input) {
```

Comment:
Let us remove this `Nullable` Annotation because it is confilct with `checkNotNull`.